vignettes/sim_fixed_design_custom.Rmd

```diff
@@ -0,0 +1,381 @@
+---
+title: "Custom Fixed Design Simulations: A Tutorial on Writing Code from the Ground Up"
+author: "Yujie Zhao and Keaven Anderson"
+output: rmarkdown::html_vignette
+bibliography: simtrial.bib
+vignette: >
+  %\VignetteIndexEntry{Custom Fixed Design Simulations: A Tutorial on Writing Code from the Ground Up}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r, message=FALSE, warning=FALSE}
+library(gsDesign2)
+library(simtrial)
+library(dplyr)
+library(gt)
+library(doFuture)
```

Comment:
It looks like you removed `library(tibble)` here. Since you explicitly call `tribble()` twice in this vignette, and also chose to add {tibble} to Suggests, you should include it here. Sorry for any confusion from my previous review. Once you add it back, you can merge.

Reply:
Please add back `library(tibble)` in this one spot. Then you can re-request a review from me, and I will approve